localisation/00_zuwc_united_localisation.csv

```diff
@@ -2848,6 +2848,14 @@ GetMonstrousUndeadNameDesc_Abomination;a horrific creation that is sewn together
 GetMonstrousUndeadNameDesc_WickerGolem;a hulking creation that is crafted from dead objects such as flesh, fur and bone and animated through an ancient death magic;;eine grobe Sch�pfung, die aus toten Materialien wie Fleisch, Fell und Knochen hergestellt und durch eine alte Todesmagie belebt wird;;;;;;;;;;;x
 EVTDESC_WCPOD_5013_wickerman;Your days in a living body come to an end. Thros offered  you a new life, giving you the opportunity to tie your soul to the wicker revenant, removing the limitations and weaknesses of flesh.;;Eure Tage in einem lebenden K�rper gehen zu Ende. Thros hat Euch ein neues Leben geschenkt und Euch die M�glichkeit gegeben, Eure Seele an den Weidenwiederg�nger zu binden und die Begrenzungen und Schw�chen des Fleisches zu �berwinden.;;;;;;;;;;;x
 EVTDESC_WCPOD_5013_wicker_golem;Your days in a living body come to an end. Thros offered you a new life, giving you the opportunity to tie your soul to the wicker golem, greatly improving your physical strength and turning you into a fearsome machine of war.;;Eure Tage in einem lebenden K�rper gehen zu Ende. Thros schenkte Euch ein neues Leben und gab Euch die M�glichkeit, Eure Seele an den Weidengolem zu binden, was Eure K�rperkraft stark erh�hte und Euch in eine furchteinfl��ende Kriegsmaschine verwandelte.;;;;;;;;;;;x
+wastewander_opinion;[wastewander.GetName] Opinion;;;;;;;;;;;;;x
+has_castle_or_city_tooltip;Has a built �YCastle�! or �YCity�!;;;;;;;;;;;;;x
```

Comment:
I'd simply put it like this:
```suggestion
has_castle_or_city_tooltip;Has a �YCastle�! or �YCity�!;;;;;;;;;;;;;x
```
or 
```suggestion
has_castle_or_city_tooltip;Owns a �YCastle�! or �YCity�!;;;;;;;;;;;;;x
```

Reply:
Done in 1106fb7623ac05e5cf0d4fb9ad421a735cd6c5f6.